lax/lichens/sciencerun0.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 import numpy as np
+from scipy.stats import binom_test
 from pax import units, configuration
+from pax.InterpolatingMap import InterpolatingMap
```

Comment:
Is there a way to *not* use the pax interpolating map?  Scipy interpolate or something?  It's another (big) dependency.